### DISCUSSION TITLE
Limiting log storage

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17819] [Q&A]
Hi all!

We have implemented Dagster in our project, it works good. We use Postgres as storage. We also have about 10 jobs that launch every minute. In order to make it clear what is happening, we use a lot of logs. The problem is that within a month Postgres began to consume a lot of memory. In this regard, I would like to know if Dagster has an option that limits the period for storing information in Postgres?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699454528006079?thread_ts=1699454528.006079&cid=C01U954MEER).*

**Comment by jamiedemaria** [comment]
Check out this discussion here about removing old entries from the DB https://github.com/dagster-io/dagster/discussions/12047
